generators/app/tasks/nextSteps.js

```diff
@@ -14,16 +14,16 @@ module.exports = function nextSteps() {
     console.log(`● All this boilerplate has already been pushed to ${this.repoUrl}`.cyan);
   }
 
-  // fabric
+  // firebase
   console.log(
-    `● Fabric and crashlytics are two ${'MANDATORY'.bold
+    `● Firebase and crashlytics are two ${'MANDATORY'.bold
       .underline} tools in every react-native project in Wolox.`.cyan
   );
   console.log('  Follow these guides in order to complete their installation:'.cyan);
```

Comment:
Is this log necessary since the implementation are already done on the generated project?

Reply:
I think having a log to remember to change the file is ok.